charts/actions-runner-controller/Chart.yaml

```
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.11.0
+version: 0.11.1
```

```suggestion
version: 0.11.0
```

A chart version change triggers a new chart release automatically on merge. That is usually ok but we're currently pending the next chart release until we cut a new controller release, as the unreleased chart contains a lot of changes that only makes sense with the next controller release. Sry for inconvenience!